### PR TITLE
Student can see topics in their course #57

### PR DESCRIPTION
**Pull request by aaronkisac**

### CODE REVIEW

File: client/src/helpers/api.js [hunk 1]
```diff
@@ -32,6 +32,11 @@ export const loginUser = async (email, password) => {
 export const getUserProfile = () => {
   return instance.get("/user/profile");
 };
+export const getLessons = (id) => {
+  var ids = id || 4;
```
Comment (llh1) [inline client/src/helpers/api.js:55]: Why do we need 4 by default?

File: client/src/helpers/api.js [hunk 2]
```diff
@@ -45,4 +45,11 @@ export const loginUser = async (email, password) => {
...
+  return instance.get("/api/courses/"+ids);
 
+export const getOrganisations = () => {
```
Comment (llh1) [inline client/src/helpers/api.js]: getOrganisations already exists in the file

File: client/src/helpers/api.js [hunk 3]
```diff
@@ -45,4 +45,11 @@ export const loginUser = async (email, password) => {
 export const getUserProfile = () => {
   return instance.get("/user/profile");
 };
+export const getLessons = (id) => {
+  var ids = id || 4;
+  return instance.get("/api/courses/"+ids);
```
Comment (NateWr) [inline client/src/helpers/api.js:56]: I think you're missing a closing bracket `}` here.

File: client/src/helpers/api.js (hunk 1 above)
Comment (aaronkisac) [reply client/src/helpers/api.js:55]: we needed firstly empty one so we used 4

File: client/src/helpers/api.js (hunk 2 above)
Comment (aaronkisac) [reply client/src/helpers/api.js]: I didnt see that sorry about that

File: client/src/helpers/api.js (hunk 3 above)
Comment (aaronkisac) [reply client/src/helpers/api.js:56]: ok how to edit any missed

Comment (NateWr) [reply client/src/helpers/api.js:56]: what

Comment (aaronkisac) [reply client/src/helpers/api.js:56]: how to solve this problem

Comment (NateWr) [reply client/src/helpers/api.js:56]: Add a `}` where it is missing.

**Review by llh1 — APPROVED** [review]

**Pull request merged**